index.html

```diff
@@ -0,0 +1,315 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+
+<!-- BEGIN ARTICLE 1 -->
+
+  <article class="product article1">
```

Comment:
Don't name classes for the sequence of the element in the HTML. If the order changes, it would lead to confusion. Name the class appropriate to the content of the element instead.